app/src/test/java/io/apicurio/registry/noprofile/rest/v1/ArtifactsResourceTest.java

```diff
@@ -636,6 +636,87 @@ public void testGetArtifactMetaDataByContent() throws Exception {
 
     }
 
+    @Test
+    public void testDeleteArtifactVersion() throws Exception {
+        String artifactContent = resourceToString("openapi-empty.json");
+
+        // Create an artifact
+        createArtifact("testDeleteArtifactVersion/EmptyAPI", ArtifactType.OPENAPI, artifactContent);
+
+        // Update the artifact 5 times
+        List<Integer> versions = new ArrayList<>();
+        for (int idx = 0; idx < 5; idx++) {
+            Integer version =
+                    given()
+                        .when()
+                            .contentType(CT_JSON)
+                            .header("X-Registry-ArtifactType", ArtifactType.OPENAPI)
+                            .pathParam("artifactId", "testDeleteArtifactVersion/EmptyAPI")
+                            .body(artifactContent.replace("Empty API", "Empty API (Update " + idx + ")"))
+                            .put("/registry/v1/artifacts/{artifactId}")
+                        .then()
+                            .statusCode(200)
+                            .body("id", equalTo("testDeleteArtifactVersion/EmptyAPI"))
+                            .body("type", equalTo(ArtifactType.OPENAPI))
+                            .extract().body().path("version");
+
+            versions.add(version);
+        }
+
+        // Delete odd artifact versions
+        for (int idx = 0; idx < 5; idx++) {
+            if (idx % 2 == 0) {
+                continue;
+            }
+
+            Integer version = versions.get(idx);
+            given()
+                .when()
+                    .pathParam("artifactId", "testDeleteArtifactVersion/EmptyAPI")
+                    .pathParam("version", version)
+                    .delete("/registry/v1/artifacts/{artifactId}/versions/{version}")
```

Comment:
This doesn't seem right, does the v1 API support version deletion?

Reply:
We definitely shouldn't be adding Delete Version to the v1 API.  Adding it to v2 makes sense, but v1 is frozen.  And I don't think v1 ever supported version delete, unless I'm mis-remembering.

Reply:
Oops, sorry, my mistake. This code was supposed to be removed

Reply:
Fixed